src/__tests__/createTheming.test.js

```diff
@@ -199,6 +199,7 @@ describe('createTheming', () => {
 
   it('merge theme from provider and prop', () => {
     const PropsChecker = withTheme(({ theme }) => {
+      expect(theme).not.toBe(lightTheme);
```

Comment:
Is this change enough to test this case with mutated theme? I'm not sure but I might be wrong ;)
See test in PR #21